Lib/test/test__colorize.py

```diff
@@ -18,38 +17,93 @@ def tearDownModule():
 
 class TestColorizeFunction(unittest.TestCase):
     @force_not_colorized
+    @unittest.skipUnless(sys.platform != "win32", "non-Windows only")
```

Comment:
Avoid double negative:
```suggestion
    @unittest.skipIf(sys.platform == "win32", "non-Windows only")
```

But why skip all the tests on Windows?

Reply:
For some reason it didn't immediately occur to me that the behavior of the function with VT sequences enabled is similar to its behavior on POSIX OSes. Reverted the changes and simplified the new test.